python/looker_sdk/rtl/api_methods.py

```diff
@@ -118,13 +117,6 @@ def _convert_query_params(
                 params[k] = json.dumps(v)
         return params
 
-    def login_user(self, user_id: int) -> "APIMethods":
-        self.auth.login_user(user_id)
-        return self
-
-    def logout(self) -> None:
-        self.auth.logout()
-
```

Comment:
another part of the actual fix - now that the generated api method classes have these methods they'd be obscured here anyway but cleaner to delete them